boards/arm/mimxrt1060_evk/mimxrt1060_evk.dts

```diff
@@ -237,7 +237,19 @@ zephyr_udc0: &usb1 {
 
 &flexpwm2_pwm3 {
 	status = "okay";
-	pinctrl-0 = <&pinmux_flexpwm2>;
+	pinctrl-0 = <&pinmux_flexpwm2_3>;
+	pinctrl-names = "default";
+};
+
+&flexpwm1_pwm0 {
+	status = "okay";
```

Comment:
Should they be enabled by default, though? This will interfere with people using those Arduino headers pins for e.g. GPIO.

Reply:
Will it? For the GPIO use case, the `gpio_configure` call should mux the pin back to a GPIO, regardless of the pin control state that the FlexPWM driver applied. There will be pin conflicts if SPI or SDHC is enabled, but that issue has always existed on these EVKs (since so many pins have their functionality overloaded).

However, if you'd prefer that I only enable these outputs as part of a sample overlay, I can update the PR.

Reply:
I think that's what most other boards do.

Reply:
Ok, I enabled them for the `pwm_api` test. I am not sure what the best place for the overlay is, because I have tested these PWM outputs with the PWM shell, but I do not see a sample that explicitly tests the PWM shell